app/scripts/lib/setupSentry.js

```diff
@@ -461,6 +46,16 @@ export default function setupSentry() {
 
   log('Initializing');
 
+  // Normally this would be awaited, but getSelf should be available by the time the report is finalized.
+  // If it's not, we still get the extensionId, and we'll just log an error for installType.
+  browser.management
+    .getSelf()
+    .then((extensionInfo) => {
+      installType = extensionInfo.installType;
```

Comment:
We might want to make this conditional:

```suggestion
      if (extensionInfo.installType) {
        installType = extensionInfo.installType;
      }
```

The MDN page suggests that this wasn't added to Firefox until v127: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/management/ExtensionInfo